lib/ui/widgets/lottie_dialog.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:flutter/material.dart';
+import 'package:lottie/lottie.dart';
+import 'package:survey_flutter_ic/utils/context_ext.dart';
+
+class LottieDialog extends StatefulWidget {
+  final Function() onAnimated;
+
+  const LottieDialog({
+    super.key,
+    required this.onAnimated,
+  });
+
+  @override
+  State<LottieDialog> createState() => _LottieDialogState();
+}
+
+class _LottieDialogState extends State<LottieDialog>
+    with TickerProviderStateMixin {
+  late final AnimationController _controller;
+
+  @override
+  void initState() {
+    _controller = AnimationController(
+      vsync: this,
+    );
+
+    super.initState();
+  }
+
+  @override
+  void dispose() {
+    _controller.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Dialog(
+      backgroundColor: Colors.black,
+      insetPadding: const EdgeInsets.all(0),
+      child: Container(
+        width: context.screenSize.width,
+        height: context.screenSize.height,
+        padding: const EdgeInsets.all(20),
```

Comment:
We can use `paddingMedium` here
```suggestion
        padding: const EdgeInsets.all(Dimensions.paddingMedium),
```